rules/okta_rules/okta_group_admin_role_assigned.py

```diff
@@ -0,0 +1,14 @@
+from panther_base_helpers import okta_alert_context
+
+
+def rule(event):
+    return event.get("eventtype", "") == "group.privilege.grant"
+
+
+def title(event):
+    # pylint: disable=W0613
+    return "Okta Admin Privileges Assigned to Group"
```

Comment:
I would include an actor who did the assigning and the group to which admin privileges were assigned.

Reply:
Unfortunately no sample logs to go off of so I'm not sure what fields would be available